### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by AlonaKaplan**
Adding .idea/ and .vscode/ to .gitignore.

### CODE REVIEW

File: pkg/controllers/interface_filter_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package controllers
```
Comment (oshoval) [inline pkg/controllers/interface_filter_test.go:1]: how this file is related to PR desc - `update gitignore` ?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 _kubevirtci/
 build/
+.idea/
+bin/
```
Comment (oshoval) [inline .gitignore:4]: where this is used?
.vscode needed as well i think

File: pkg/controllers/interface_filter_test.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "kubevirt.io/api/core/v1"
```
Comment (oshoval) [inline pkg/controllers/interface_filter_test.go:6]: insert line please

**Review by oshoval — COMMENTED** [review]
Thanks

Didnt review all yet
maybe this is even related to #26 so better to review it there

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,5 @@
...
+cover.out
```
Comment (oshoval) [inline .gitignore:5]: should be part of #26 imo as it present the file

does it also present bin/ ? if so same deal

File: .gitignore (hunk 2 above)
Comment (AlonaKaplan) [reply .gitignore:4]: Regarding the `bin`, I"m not sure maybe by the operator-sdk.
I have `ls -l bin/
controller-gen  k8s/            setup-envtest`
But I think makes sesne anyway to have it in the .gitignore.
Will add .vscode

**Comment by oshoval** [comment]
Do we know why one of them has / and the other doesnt ?
maybe worth to just touch a file in both formats and see if both works to use same style for all folders ?
(also the ones that already exists)

**Review by oshoval — APPROVED** [review]
Thanks

There is this note
https://github.com/AlonaKaplan/kubesecondarydns/pull/27#issuecomment-1325358737
but feel free to merge if you prefer the current way

**Comment by oshoval** [comment]
Please add in the PR desc that those IDE folders are added to be more developer friendly maybe

**Pull request merged**